validation-test/compiler_crashers_fixed/28707-false-encountered-error-in-diagnostic-text.swift

```diff
@@ -6,7 +6,7 @@
 // See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 
 // REQUIRES: asserts
```

Comment:
Please remove the REQUIRES line -- once a crasher is fixed, it no longer requires asserts to be on to pass